Assets/MixedRealityToolkit.Services/InputSimulation/BaseInputSimulationService.cs

```diff
@@ -123,16 +123,11 @@ protected SimulatedHand GetOrAddHandDevice(Handedness handedness, HandSimulation
             }
 
             System.Type controllerType = simulationMode == HandSimulationMode.Gestures ? typeof(SimulatedGestureHand) : typeof(SimulatedArticulatedHand);
-            if (controller == null)
-            {
-                Debug.LogError($"Failed to create {controllerType} controller");
-                return null;
-            }
-
-            if (!controller.SetupConfiguration(controllerType))
+            
+            if (controller == null || !controller.Enabled)
```

Comment:
This block is repeated in every (?) manager class. Can it be placed in a base class and eliminate the duplication?

Reply:
I'll take a look at this today

Reply:
Okay, looking at this, I think that yes, we can do some de-duping here. I already have a few follow-up PRs to this one, so I'm going to propose this change is further investigated / implemented as part of an additional PR, as it's slightly outside the scope of this one in my mind.

https://github.com/microsoft/MixedRealityToolkit-Unity/issues/7416